Corona-Warn-App/src/test/java/de/rki/coronawarnapp/submission/DaysSinceOnsetOfSymptomsVectorDeterminatorTest.kt

```diff
@@ -0,0 +1,141 @@
+package de.rki.coronawarnapp.submission
+
+import org.junit.Assert
+import org.junit.Test
+
+class DaysSinceOnsetOfSymptomsVectorDeterminatorTest {
+
+    @Test
+    fun positive_exactDate_Yesterday() {
+        val daysAgo = 1
+        Assert.assertArrayEquals(
+            intArrayOf(1, 0, -1, -2, -3, -4, -5, -6, -7, -8, -9, -10, -11, -12, -13),
+            DaysSinceOnsetOfSymptomsVectorDeterminator().determine(
+                Symptoms(
+                    Symptoms.StartOf.Date(System.currentTimeMillis() - 1000 * 3600 * (24 * daysAgo + 2)),
```

Comment:
Ideally we would not use variable time value (`currentTimeMillis`) in tests, but pass a fixed/mocked time sources.